app/src/main/java/com/movies/presentation/activity/MoviesActivity.kt

```diff
@@ -13,6 +15,10 @@ class MoviesActivity : AppCompatActivity() {
         binding = ActivityMoviesBinding.inflate(layoutInflater)
         installSplashScreen()
         setContentView(binding.root)
+
+        supportFragmentManager.beginTransaction().add(
+            R.id.nav_host_fragment, HomeFragment(), "home"
```

Comment:
you can just remove third parameter "home". do you use it anywhere?

Reply:
done, removed unused parameter